app/src/main/java/com/android/wildex/ui/social/FriendScreen.kt

```diff
@@ -732,8 +743,10 @@ fun CurrentUserFriendScreenContent(
     setSelectedTab: (String) -> Unit,
     friendScreenViewModel: FriendScreenViewModel,
     state: FriendsScreenUIState,
-    onProfileClick: (Id) -> Unit
+    onProfileClick: (Id) -> Unit,
+    userIndex: UserIndex
 ) {
+  UserSearchBar(userIndex = userIndex, onResultClick = onProfileClick)
```

Comment:
Part of the the exclude suggestion in the general comment

```suggestion
  UserSearchBar(userIndex = userIndex, onResultClick = onProfileClick, currentUserId = currentUserId)
```